integration/tests/basic/rpc/test_rpc_get_transaction.py

```diff
@@ -258,6 +284,7 @@ def test_neon_get_transaction_by_sender_nonce_negative(self, params, error_code,
         assert error_code == response["error"]["code"]
         assert error_message == response["error"]["message"]
 
+    # Geth returns code 32601
```

Comment:
any issue created?

Reply:
This comment is not necessary because it's for neon_* operation. Removing it ✅